src/main/java/dave/Ui.java

```diff
@@ -0,0 +1,122 @@
+package dave;
+
+import java.util.Scanner;
+
+import dave.tasks.Task;
+
+public class Ui {
+    /** The user input scanner. */
+    private Scanner sc;
+
+    /** Name of the chatbot, Dave. */
+    static final String NAME = "Dave";
+    /** Horizontal line divider for output. */
+    static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+
+    /** 
+     * Creates new UI object.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Shows the horizontal line divider in output.
+     */
+    public void showHorizontalLine() {
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    /**
+     * Welcomes the user with a message.
+     */
+    public void greet() {
+        System.out.println(String.format("%s\n%s: Nice to meet you, I'm the ever-helpful %s!\nHow may I be of service today?\n%s",
+        HORIZONTAL_LINE, NAME, NAME, HORIZONTAL_LINE));
+        return;
+    }
```

Comment:
Just a suggestion, maybe sayHi() would also be a good name since you used sayBye() below